ote/src/cljs/ote/app/routes.cljs

```diff
@@ -149,33 +149,34 @@
                      ;; Remove potentially sensitive arguments from analytics script reporting
                      win-location (if-let [opt-out-page (#{:register :reset-password :confirm-email} (:page navigation-data))]
                                     (str "/" (name opt-out-page))
-                                    win-location)]
-
-                 (if (or (requires-authentication? app)
-                         (requires-transit-authority? app)
-                         (requires-admin? app))
+                                    win-location)
+                     not-authorized? (or (requires-authentication? app)
```

Comment:
Negation and inversion should be avoided in naming for better readability when referring in complex conditions.
`not-authorized?` ==> `authorized?`